lib/Serialization/Deserialization.cpp

```diff
@@ -4713,17 +4715,20 @@ class swift::TypeDeserializer {
                                                   noescape,
                                                   // SWIFT_ENABLE_TENSORFLOW
                                                   throws,
-                                                  differentiable);
+                                                  rawDiffKind);
+      diffKind = DifferentiabilityKind(rawDiffKind);
+      
```

Comment:
```suggestion
```

Reply:
Ahh how were you able to tell there was extra whitespace there?

Reply:
Hmm, it's actually not about whitespace, but about the extra line.

Reply:
Oh okay will remove it